### PR TITLE
#40-idoit-install-incorrectly-detects-mariadb-version

### PR DESCRIPTION
**Pull request by MichaelOv**
Altered mariadb/mysql version check see #40
Minor fixes

### CODE REVIEW

File: idoit-install [hunk 1]
```diff
@@ -1350,7 +1355,7 @@ EOF
         "$a2_en_mod" setenvif || abort "Unable to enable Apache module setenvif"
         log "Let every user read the logs"
         chmod 755 /var/log/apache2 || abort "Unable to change permissions"
-        chmod 664 /var/log/apache2/* || abort "Unable to change permissions"
+        chmod 664 /var/log/apache2/ || abort "Unable to change permissions"
```
Comment (bheisig) [inline idoit-install:1358]: `664` on a directory seems not right

File: idoit-install [hunk 2]
```diff
@@ -1505,15 +1510,16 @@ EOF
     if [[ -d "/var/log/mysql" ]]; then
         log "Let every user read the logs"
         chmod 755 /var/log/mysql
-        chmod 664 /var/log/mysql/*
+        chmod 664 /var/log/mysql/
     fi
 
-    unitctl "start" "$MARIADB_UNIT"
```
Comment (bheisig) [inline idoit-install:1511]: Are you sure MariaDB is already running?

File: idoit-install (hunk 1 above)
Comment (MichaelOv) [reply idoit-install:1358]: true i accidentially removed the asterisk :<

File: idoit-install (hunk 2 above)
Comment (MichaelOv) [reply idoit-install:1511]: true, i will revert it

**Review by bheisig — APPROVED** [review]

**Pull request merged**